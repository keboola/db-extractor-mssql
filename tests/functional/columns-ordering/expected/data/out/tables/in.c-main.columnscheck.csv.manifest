{"destination":"in.c-main.columnsCheck","incremental":false,"metadata":[{"key":"KBC.name","value":"sales"},{"key":"KBC.sanitizedName","value":"sales"},{"key":"KBC.schema","value":"dbo"},{"key":"KBC.catalog","value":"test"},{"key":"KBC.type","value":"BASE TABLE"}],"column_metadata":{"createdat":[{"key":"KBC.datatype.type","value":"varchar"},{"key":"KBC.datatype.nullable","value":false},{"key":"KBC.datatype.basetype","value":"STRING"},{"key":"KBC.datatype.length","value":"64"},{"key":"KBC.sourceName","value":"createdat"},{"key":"KBC.sanitizedName","value":"createdat"},{"key":"KBC.primaryKey","value":true},{"key":"KBC.uniqueKey","value":false},{"key":"KBC.ordinalPosition","value":6}],"categorygroup":[{"key":"KBC.datatype.type","value":"text"},{"key":"KBC.datatype.nullable","value":true},{"key":"KBC.datatype.basetype","value":"STRING"},{"key":"KBC.datatype.length","value":"2147483647"},{"key":"KBC.sourceName","value":"categorygroup"},{"key":"KBC.sanitizedName","value":"categorygroup"},{"key":"KBC.primaryKey","value":false},{"key":"KBC.uniqueKey","value":false},{"key":"KBC.ordinalPosition","value":12}],"sku":[{"key":"KBC.datatype.type","value":"text"},{"key":"KBC.datatype.nullable","value":true},{"key":"KBC.datatype.basetype","value":"STRING"},{"key":"KBC.datatype.length","value":"2147483647"},{"key":"KBC.sourceName","value":"sku"},{"key":"KBC.sanitizedName","value":"sku"},{"key":"KBC.primaryKey","value":false},{"key":"KBC.uniqueKey","value":false},{"key":"KBC.ordinalPosition","value":5}],"zipcode":[{"key":"KBC.datatype.type","value":"text"},{"key":"KBC.datatype.nullable","value":true},{"key":"KBC.datatype.basetype","value":"STRING"},{"key":"KBC.datatype.length","value":"2147483647"},{"key":"KBC.sourceName","value":"zipcode"},{"key":"KBC.sanitizedName","value":"zipcode"},{"key":"KBC.primaryKey","value":false},{"key":"KBC.uniqueKey","value":false},{"key":"KBC.ordinalPosition","value":4}],"userstate":[{"key":"KBC.datatype.type","value":"text"},{"key":"KBC.datatype.nullable","value":true},{"key":"KBC.datatype.basetype","value":"STRING"},{"key":"KBC.datatype.length","value":"2147483647"},{"key":"KBC.sourceName","value":"userstate"},{"key":"KBC.sanitizedName","value":"userstate"},{"key":"KBC.primaryKey","value":false},{"key":"KBC.uniqueKey","value":false},{"key":"KBC.ordinalPosition","value":11}]},"columns":["createdat","categorygroup","sku","zipcode","userstate"]}
