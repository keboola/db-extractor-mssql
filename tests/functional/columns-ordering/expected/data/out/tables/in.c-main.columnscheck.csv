"2013-09-23 22:38:29","Electronics","ZD111318","39153","Mississippi"
"2013-09-23 22:38:30","Electronics","ZD111402","89124","Nevada"
"2013-09-23 22:38:31","Jewelry","ZD111483","21020","Maryland"
"2013-09-23 22:38:32","Electronics","ZD111395","18501","Pennsylvania"
"2013-09-23 22:38:33","Jewelry","ZD111451","44127","Ohio"
"2013-09-23 22:38:34","Jewelry","ZD111471","21219","Maryland"
"2013-09-23 22:38:35","Books","ZD111228","85083","Arizona"
"2013-09-23 22:38:36","Electronics","ZD111340","25428","West Virginia"
"2013-09-23 22:38:37","Women","ZD111595","91328","California"
"2013-09-23 22:38:38","Women","ZD111577","28455","North Carolina"
"2013-09-23 22:38:39","Books","ZD111274","07974","New Jersey"
"2013-09-23 22:38:40","Books","ZD111146","48359","Michigan"
"2013-09-23 22:38:41","Men","ZD111542","02764","Massachusetts"
"2013-09-23 22:38:42","Women","ZD111603","74156","Oklahoma"
"2013-09-23 22:38:43","Jewelry","ZD111450","92806","California"
"2013-09-23 22:38:44","Men","ZD111548","21029","Maryland"
"2013-09-23 22:38:45","Women","ZD111577","27837","North Carolina"
"2013-09-23 22:38:46","Jewelry","ZD111438","80615","Colorado"
"2013-09-23 22:38:47","Electronics","ZD111296","63126","Missouri"
"2013-09-23 22:38:48","Electronics","ZD111287","14551","New York"
"2013-09-23 22:38:49","Men","ZD111532","47351","Indiana"
"2013-09-23 22:38:50","Men","ZD111535","56372","Minnesota"
"2013-09-23 22:38:51","Books","ZD111227","48836","Michigan"
"2013-09-23 22:38:52","Books","ZD111113","96155","California"
"2013-09-23 22:38:53","Electronics","ZD111391","66027","Kansas"
"2013-09-23 22:38:54","Books","ZD111232","54643","Wisconsin"
"2013-09-23 22:38:55","Jewelry","ZD111431","71291","Louisiana"
"2013-09-23 22:38:56","Electronics","ZD111287","60630","Illinois"
"2013-09-23 22:38:57","Books","ZD111135","34231","Florida"
"2013-09-23 22:38:58","Books","ZD111201","48210","Michigan"
"2013-09-23 22:38:59","Jewelry","ZD111423","91328","California"
"2013-09-23 22:39:00","Jewelry","ZD111512","91386","California"
"2013-09-23 22:39:01","Electronics","ZD111374","22940","Virginia"
"2013-09-23 22:39:02","Jewelry","ZD111455","92119","California"
"2013-09-23 22:39:03","Women","ZD111605","33755","Florida"
"2013-09-23 22:39:04","Books","ZD111243","28289","North Carolina"
"2013-09-23 22:39:05","Books","ZD111185","07506","New Jersey"
"2013-09-23 22:39:06","Books","ZD111281","92518","California"
"2013-09-23 22:39:07","Books","ZD111253","70049","Louisiana"
"2013-09-23 22:39:08","Books","ZD111238","18951","Pennsylvania"
"2013-09-23 22:39:09","Books","ZD111184","76558","Texas"
"2013-09-23 22:39:10","Books","ZD111161","45403","Ohio"
"2013-09-23 22:39:11","Jewelry","ZD111455","90034","California"
"2013-09-23 22:39:12","Books","ZD111118","72379","Arkansas"
"2013-09-24 08:27:31","Jewelry","ZD111451","52042","Iowa"
"2013-09-24 08:27:34","Jewelry","ZD111426","40285","Kentucky"
"2013-09-24 08:27:35","Jewelry","ZD111500","74015","Oklahoma"
"2013-09-24 08:27:36","Electronics","ZD111392","84402","Utah"
"2013-09-24 08:27:37","Jewelry","ZD111524","91507","California"
"2013-09-24 08:27:38","Electronics","ZD111407","23694","Virginia"
"2013-09-24 08:27:39","Jewelry","ZD111436","16046","Pennsylvania"
"2013-09-24 08:27:40","Jewelry","ZD111424","61880","Illinois"
"2013-09-24 08:27:41","Electronics","ZD111303","92038","California"
"2013-09-24 08:27:42","Books","ZD111138","28217","North Carolina"
"2013-09-24 08:27:43","Books","ZD111180","63132","Missouri"
"2013-09-24 08:27:44","Books","ZD111266","39402","Mississippi"
"2013-09-24 08:27:45","Electronics","ZD111292","02564","Massachusetts"
"2013-09-24 08:27:46","Books","ZD111278","33309","Florida"
"2013-09-24 08:27:47","Books","ZD111127","14215","New York"
"2013-09-24 08:27:48","Books","ZD111183","97707","Oregon"
"2013-09-24 08:27:49","Jewelry","ZD111518","33326","Florida"
"2013-09-24 08:27:50","Books","ZD111142","33339","Florida"
"2013-09-24 08:27:51","Electronics","ZD111308","33454","Florida"
"2013-09-24 08:27:52","Electronics","ZD111369","29169","South Carolina"
"2013-09-24 08:27:53","Electronics","ZD111316","48458","Michigan"
"2013-09-24 08:27:55","Electronics","ZD111364","67202","Kansas"
"2013-09-24 08:27:56","Jewelry","ZD111458","94507","California"
"2013-09-24 08:27:57","Women","ZD111598","71294","Louisiana"
"2013-09-24 08:27:58","Electronics","ZD111329","72628","Arkansas"
"2013-09-24 08:27:59","Electronics","ZD111316","66046","Kansas"
"2013-09-24 08:28:00","Books","ZD111181","22109","Virginia"
"2013-09-24 08:28:01","Women","ZD111592","63127","Missouri"
"2013-09-24 08:28:02","Electronics","ZD111331","48911","Michigan"
"2013-09-24 08:28:03","Books","ZD111177","80219","Colorado"
"2013-09-24 08:28:04","Books","ZD111167","32606","Florida"
"2013-09-24 08:28:05","Jewelry","ZD111478","48086","Michigan"
"2013-09-24 08:28:06","Jewelry","ZD111425","36550","Alabama"
"2013-09-24 08:28:07","Books","ZD111236","35957","Alabama"
"2013-09-24 08:28:08","Books","ZD111190","33128","Florida"
"2013-09-24 08:28:09","Women","ZD111605","95887","California"
"2013-09-24 08:28:10","Books","ZD111246","62034","Illinois"
"2013-09-24 08:28:11","Electronics","ZD111400","19128","Pennsylvania"
"2013-09-24 08:28:12","Books","ZD111166","93923","California"
"2013-09-24 08:28:13","Jewelry","ZD111528","69358","Nebraska"
"2013-09-24 08:28:14","Books","ZD111164","90223","California"
"2013-09-24 08:28:15","Books","ZD111196","98686","Washington"
"2013-09-24 08:28:16","Books","ZD111118","32724","Florida"
"2013-09-24 08:28:17","Electronics","ZD111394","76258","Texas"
"2013-09-24 08:28:18","Electronics","ZD111292","63303","Missouri"
"2013-09-24 08:28:19","Books","ZD111141","15270","Pennsylvania"
"2013-09-24 08:28:20","Electronics","ZD111331","11447","New York"
"2013-09-24 08:28:21","Electronics","ZD111338","75015","Texas"
"2013-09-24 08:28:22","Jewelry","ZD111460","13777","New York"
"2013-09-24 08:28:23","Electronics","ZD111313","30025","Georgia"
"2013-09-24 08:28:24","Jewelry","ZD111446","46346","Indiana"
"2013-09-24 08:28:25","Jewelry","ZD111466","55375","Minnesota"
"2013-09-24 08:28:26","Books","ZD111178","11366","New York"
"2013-09-24 08:28:27","Men","ZD111560","17260","Pennsylvania"
"2013-09-24 08:28:28","Books","ZD111132","92124","California"
"createdat","group","sku","zipcode","userstate"
